### PR TITLE
gphoto2: bumped to 2.5.28

### PR DESCRIPTION
**Pull request by Vouivre**
## Notes for 2.5.28

2.5.28 Can't be build without `libexif`. It has been fixed and it will be possible to build without
`libexif` with the next release. Until the next release a patch is needed. Details in this issue:

https://github.com/gphoto/gphoto2/issues/479

I chosed to use the patch and execute `autoreconf`. So I add `autoconf` as a dependecy. 
Another possibility would be to create the `configure` script on my system and to add it in
the package instead of the patch. But I'm not sure it would be adapted to all system. If you have another
suggestion, I can adapt the PR. 

I have no idea if there are standard paramters to `autoreconf` like `-i`. I can also add them. 

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/gphoto2/version
/var/db/kiss/installed/gphoto2/sources
/var/db/kiss/installed/gphoto2/post-install
/var/db/kiss/installed/gphoto2/manifest
/var/db/kiss/installed/gphoto2/files/libexif.patch
/var/db/kiss/installed/gphoto2/files/
/var/db/kiss/installed/gphoto2/depends
/var/db/kiss/installed/gphoto2/checksums
/var/db/kiss/installed/gphoto2/build
/var/db/kiss/installed/gphoto2/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/gphoto2.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/gphoto2/test-hook.sh
/usr/share/doc/gphoto2/
/usr/share/doc/
/usr/share/
/usr/bin/gphoto2
/usr/bin/
/usr/
```

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/gphoto2/depends [hunk 1]
```diff
@@ -1,2 +1,3 @@
+autoconf
```
Comment (dilyn-corner) [inline community/gphoto2/depends:1]: Should be a make dep

**Review by dilyn-corner — CHANGES_REQUESTED** [review]

**Comment by dilyn-corner** [comment]
As for your comment:
> I have no idea if there are standard paramters to autoreconf like -i. I can also add them.

There are some standard things that are done as part of using autotools. Specifically, `autoreconf` calls a flurry of packages which run repeatedly until an entire, functioning build system (defined as "buildable by `gmake; gmake install`") is created. Some examples include `autopoint` and `libtoolize`, as required. They do things like moving `m4` files, using `libtool`, running `gettext`, etc.

Style-wise in KISS, the assumed default is running `autoreconf -sfi`, though this is only in place of running `./autogen.sh` usually, as this accomplishes a roughly similar thing without doing some extra shenanigans that might be in the autogen script.

Because you only need to regenerate `configure` from `configure.ac`, running `autoconf` should be sufficient, and is one of the things called by `autoreconf`. So long as it builds (which it should), using `autoconf` instead of `autoreconf -sfi` is fine by me (though it should result in the same thing).

File: community/gphoto2/depends (hunk 1 above)
Comment (Vouivre) [reply community/gphoto2/depends]: Of course! Done.

**Comment by Vouivre** [comment]
After reading your second comment, I changed `autoreconf` to `autoconf`. 

Thank you for taking time to write such a detailed comment!

**Comment by dilyn-corner** [comment]
It's what I do!

**Pull request merged**